src/meltano/core/tracking/plugins.py

```diff
@@ -0,0 +1,57 @@
+"""Tracking plugin context for the Snowplow tracker."""
+from __future__ import annotations
+
+import uuid
+
+from snowplow_tracker import SelfDescribingJson
+from structlog.stdlib import get_logger
+
+from meltano.core.plugin.project_plugin import ProjectPlugin
+from meltano.core.utils import hash_sha256
+
+logger = get_logger(__name__)
+
+PLUGINS_CONTEXT_SCHEMA = "iglu:com.meltano/plugins_context/jsonschema"
+PLUGINS_CONTEXT_SCHEMA_VERSION = "1-0-0"
+
+
+def _from_plugin(plugin: ProjectPlugin, cmd: str) -> dict:
+    return {
+        "category": str(plugin.type),
+        "name_hash": hash_sha256(plugin["info"].get("name")),
+        # TODO: what if its "original" should be we hashing namespace and sending that as well?
+        "variant_name_hash": hash_sha256(plugin["info"].get("variant")),
+        "pip_url_hash": hash_sha256(plugin.formatted_pip_url),
+        # TODO: this is not the right field, but parent "name" isn't available in the plugin context
+        "parent_name_hash": hash_sha256(plugin.parent.executable),
```

Comment:
@aaronsteers Should I be grabbing `plugin.parent.definition["name"]` for this ?


![Screen Shot 2022-06-05 at 1 19 58 PM](https://user-images.githubusercontent.com/333354/172064797-51af51fb-ed2f-4640-af32-7e127046dbc2.png)

Reply:
(fyi in more recent commit i do just that)